modules/View.js

```diff
@@ -86,12 +87,26 @@
      * @method render
      */
     render: function() {
-      this.unplug();
```

Comment:
This and ListView seem to have a lot of common code - can we reuse that somehow?